index.html

```diff
@@ -0,0 +1,499 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/e252d07dd4.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Heatmap</title>
+  </head>
+
+  <body>
+    <nav class="max-w nav-container">
+      <div class="griditems1 boxes color-black">1</div>
+      <div class="griditems2 boxes d-grid fifth-color">Article</div>
+      <div class="griditems3 d-grid fifth-color">Guide</div>
+      <div class="griditems4 d-grid fifth-color">Books</div>
+      <div class="griditems5 d-grid fifth-color">Events</div>
+      <div class="griditems6 d-grid fifth-color">Membership</div>
+      <div class="griditems7 d-grid fifth-color">More</div>
+      <div class="griditems8 boxes">
+        <input type="search" placeholder="Search articles" />
+      </div>
+    </nav>
+
+    <!-- featured articles -->
+    <div class="featured flex-col">
```

Comment:
- [OPTIONAL] You can consider the use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document